### PR TITLE
Documentation for transferring data between CKAN instances

### PR DESCRIPTION
**Pull request by brucebolt**

### CODE REVIEW

File: source/manual/data-gov-uk-supporting-ckan.html.md [hunk 1]
```diff
@@ -179,3 +179,97 @@ to purge the queues used in the various stages of harvesting
...
+
+To administer a remote instance, you will require a sysadmin account, your username and an API key (obtained through the web interface).  Append the remote URL and API key to each ckanapi request, in place of the `-c` flag:
+
+```
+ckanapi -r URL -a API_KEY ...
+```
+
+#### Dumping data
+
+Active data (i.e. everything that is publicly available) can be retrieved through the API.  To dump data that is not available publicly, you must using an API key belonging to a sysadmin user.
```
Comment (andrewgarner) [inline source/manual/data-gov-uk-supporting-ckan.html.md]: "you must use"

File: source/manual/data-gov-uk-supporting-ckan.html.md [hunk 2]
```diff
@@ -179,3 +179,97 @@ to purge the queues used in the various stages of harvesting
...
+
+Retrieve the user data from Drupal (this is a nightly dumped file):
+
+```
+rsync -L --progress co@co-prod3.dh.bytemark.co.uk:/var/lib/ckan/ckan/dumps_with_private_data/drupal_users_table.csv.gz DRUPAL_USER_DUMP.csv.gz
+```
+
+#### Data schema migration
+
+The data schema for the two CKAN instances may not match.  Therefore a conversion is required to allow the dump to be imported to the new CKAN installation.  Scripts are available in `ckanext-datagovuk` to enable this.
```
Comment (andrewgarner) [inline source/manual/data-gov-uk-supporting-ckan.html.md]: How about we link to https://github.com/alphagov/ckanext-datagovuk?

File: source/manual/data-gov-uk-supporting-ckan.html.md (hunk 1 above)
Comment (brucebolt) [reply source/manual/data-gov-uk-supporting-ckan.html.md]: Change made and rebased

File: source/manual/data-gov-uk-supporting-ckan.html.md (hunk 2 above)
Comment (brucebolt) [reply source/manual/data-gov-uk-supporting-ckan.html.md]: Change made and rebased

**Comment by brucebolt** [comment]
That's a good point.  I'll move this to a separate manual page.

**Comment by brucebolt** [comment]
Moved this into a new manual page

**Pull request merged**